src/common/config_parser.cpp

```diff
@@ -634,22 +639,20 @@ void ConfigParser::parseOptions(int argc, char** argv, bool doValidate) {
     auto config = loadConfigFiles(configPaths);
     // combine loaded options with the main YAML config
     cli.overwriteDefault(config);
-    config_ = cli.getConfig();
   }
 
-  if(has("interpolate-env-vars")) {
+  if(get<bool>("interpolate-env-vars")) {
```

Comment:
Don't these need the optional default value? `get<bool>("...", false)`?

Reply:
This has a default value of "false", since we are inside the config parser which sees the option definitions, this is safe.